src/utils.jl

```diff
@@ -338,3 +338,49 @@ function show_stackloc(io::IO, frame)
     end
 end
 show_stackloc(frame) = show_stackloc(stdout, frame)
+
+# Pretty printing of stacktraces and errors with Frame
+function Base.StackTraces.StackFrame(frame::Frame)
+    if frame.framecode.scope isa Method
+        method = frame.framecode.scope
+        method_args = something.(frame.framedata.locals[1:frame.framecode.scope.nargs])
+        atypes = Tuple{typeof.(method_args)...}
+        sig = frame.framecode.scope.sig
+        (ti, lenv) = ccall(:jl_type_intersection_with_env, Any, (Any, Any), atypes, sig)
```

Comment:
We already have the `sparams` but I need them as an `SimpleVector`. Is there another way to do this then just call the intersection again?

Reply:
Yes, but with `framedata.sparams` instead of `framedata.locals`